cli/lib/kontena/cli/stacks/upgrade_command.rb

```diff
@@ -220,6 +218,7 @@ def run_deploys(deployable_stacks)
       deployable_stacks.each do |deployable_stack|
         Kontena.run!(['stack', 'deploy', deployable_stack])
       end
+      set_env_variables(stack_name, current_grid)
```

Comment:
This probably doesn't hurt here, but I'm also not entirely sure what it's doing... Voodoo code with spooky action at a distance, or a talisman better just left out?

Reply:
![image](https://user-images.githubusercontent.com/224971/34106055-6c21c2c4-e3ff-11e7-8e5f-baddb9c9e0d7.png)